lightly/active_learning/scorers/scorer.py

```diff
@@ -8,5 +8,13 @@ class Scorer():
     def __init__(self, model_output):
         self.model_output = model_output
 
-    def _calculate_scores(self) -> Dict[str, np.ndarray]:
+    def calculate_scores(self) -> Dict[str, np.ndarray]:
+        """Calculates and returns the active learning scores
+
+        Which scores are calculated depends on the implementation
+        of this parent class by the child classes.
+        Returns:
```

Comment:
This function does not return anything so I'd remove the second part of your comment. IMO the following suffices:

"Calculates and returns active learning scores in a dictionary."